src/main/java/org/mockbukkit/mockbukkit/world/ChunkDataMock.java

```diff
@@ -156,6 +158,16 @@ public byte getData(int x, int y, int z)
 		return this.getTypeAndData(x, y, z).getData();
 	}
 
+	@Override
+	public int getHeight(@NotNull HeightMap heightMap, @Range(from = 0L, to = 15L) int x, @Range(from = 0L, to = 15L) int z)
+	{
+		Preconditions.checkArgument(heightMap != null, "HeightMap cannot be null");
+		Preconditions.checkArgument(x >= 0 && x <= 15 && z >= 0 && z <= 15, "Cannot get height outside of a chunks bounds, must be between 0 and 15, got x: %s, z: %s", x, z);
+
+		// TODO Auto-generated method stub
```

Comment:
Is this intended?

There's no implementation

Reply:
Yes, i saw that the method check argument existed so i just copied it over.